ApollosProject/apollos-apps
# Update index.js

## ✏️ Description

This PR updates the Date selection component to use the newer `PressableChip` component in place of the old one. This is to resolve an issue where the label color is not properly updating when flipping between light and dark mode.

## 🔬 To Test
1. Run in storybook
2. Check onboarding screens for birthday input

## 📸 Screenshots
| Old | Fixed |
| --- | --- | 
| ![IMG_401AC83484A4-1](https://user-images.githubusercontent.com/45076058/219798972-0ea17df0-d034-43ea-b687-6b91d06b9080.jpeg) | ![CleanShot 2023-02-17 at 16 41 35](https://user-images.githubusercontent.com/45076058/219799216-0f5c9721-58b7-45ac-b778-b71d8231b8e3.png) |


## ⚠️ To-do Before Merge
n/a

## 📝 Checklist
